.reuse/dep5

```diff
@@ -0,0 +1,56 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: firstrunwizard
+Upstream-Contact: Nextcloud <info@nextcloud.com>
+Source: https://github.com/nextcloud/firstrunwizard
+
+Files: .gitattributes .github/* .editorconfig babel.config.js .php-cs-fixer.dist.php package-lock.json package.json composer.json composer.lock */composer.json */composer.lock webpack.js stylelint.config.js .eslintrc.js .gitignore .jshintrc .l10nignore action/.gitignore action/package.json action/package-lock.json action/dist/index.js tests/* psalm.xml cypress/.env cypress/.eslintrc.js cypress/docker-compose.yml cypress/plugins/index.js cypress/tsconfig.json cypress.config.ts vendor-bin/*/composer.json stylelint.config.cjs vendor-bin/*/composer.lock .tx/config webpack.config.js tsconfig.json vite.config.ts vite.config.js js/vendor.LICENSE.txt krankerl.toml .npmignore cypress/fixtures/* postcss.config.js cypress/dockerNode.ts jest.config.js cypress.config.js relativeci.config.js .nextcloudignore .devcontainer/devcontainer.json .scrutinizer.yml .stylelintrc.js .tx/backport
```

Comment:
Is this some kind of template? I think it contains a lot of files that should rather have proper license tags, no?

Reply:
yes, I c&p from existing repos. So yes, some files for sure could have proper headers

Reply:
could be done in a follow up though

Reply:
Added / replaced old license tags on test files and config files to resolve this.
Also updated all workflows from templates to include the headers, see last two commits.